build-release-tools/post_test.sh

```diff
@@ -381,18 +381,22 @@ post_test_docker() {
     clean_all_containers
     cd $RackHDDir/docker 
     #if clone file name is not repo name, this scirpt should be edited.
-    while read -r LINE; do
-        cp docker-compose-mini.yml docker-compose-mini.yml.bak
-        for repo_tag in $LINE; do
-            repo=${repo_tag%:*}
-            sed -i "s#rackhd/${repo}.*#rackhd/${repo_tag}#g" docker-compose-mini.yml
-        done
-        docker-compose -f docker-compose-mini.yml pull --ignore-pull-failures
-        docker-compose -f docker-compose-mini.yml up -d
-        mv docker-compose-mini.yml.bak docker-compose-mini.yml
-        waitForAPI
-        clean_all_containers
-    done < $buildRecord
+    # build recorde is a text file with two lines, a daily build_record is like:
+    ####################################
+    # on-core:$tag on-tasks:$tag ....  #
+    # on-core:nightly on-tasks:nightly #
+    ####################################
+    image_list=$(head -n 1 $buildRecord)
+    cp docker-compose-mini.yml docker-compose-mini.yml.bak
+    for repo_tag in $image_list; do
+        repo=${repo_tag%:*}
+        sed -i "s#rackhd/${repo}.*#rackhd/${repo_tag}#g" docker-compose-mini.yml
+    done
+    docker-compose -f docker-compose-mini.yml pull --ignore-pull-failures
+    docker-compose -f docker-compose-mini.yml up -d
```

Comment:
@keedya  @changev ,sorry to jump in,  I guess to pull the other 3rd party images like mongo ?